Added Changelog for Feb and March (#5653)
codesandbox/codesandbox-client: packages/homepage/content/changelog/4-0-1/index.md

---
banner: ./4-0-1.jpg
title: v4.0.1
slug: Custom Artwork
date: 2021-02-23
description: Add custom thumbnails to your sandboxes
---

#### Features

> 1. Anti Phishing banner 

Reviewer: ```suggestion
1. Add a warning banner to sandboxes that are more likely to be phishing
```

2. Keyboard traversing in file tree [@jyash97](https://github.com/codesandbox/codesandbox-client/pull/5505)

> 3. Logging for errors when installing private package 

Reviewer: ```suggestion
3. More helpful error messages for errors while installing a private package
```

> 4. New Over-mind typing 

Reviewer: ```suggestion
4. Improvements in our state management 
```

5. Custom Cover Images for sandboxes 

#### Fixes

> - Export ordering when re-exporting an import 

Reviewer: ```suggestion
- Bundler: fix the export ordering when you're re-exporting an import
```

> - Yarn lockfile was updated [@NullVoxPopuli](https://github.com/codesandbox/codesandbox-client/pull/5501)

Reviewer: This is more of an internal maintenance, so I would not put this one in explicitly.

```suggestion
```
